doc/man5/flux-config-bootstrap.rst

```diff
@@ -29,6 +29,11 @@ expected to bootstrap from an identical config file.
 KEYWORDS
 ========
 
+enable_ipv6
+   (optional) Boolean value for enabling IPv6.  By default only IPv4 is
+   enabled.  Note that setting this to true prevents binding to a named
+   interface that only supports IPv4.
```

Comment:
Do you feel the current behavior of zeromq is a bug? If so, maybe we should open an issue against zeromq and reference that issue here (Saying "on versions of zeromq affected by the following issue...")

Reply:
Not sure.  I think maybe I just didn't anticipate this particular outcome?  The extent of 0MQ's documentation appears to be (From zmq_setsockopt(3)):
```
   ZMQ_IPV6: Enable IPv6 on socket
       Set the IPv6 option for the socket. A value of 1 means IPv6 is enabled
       on the socket, while 0 means the socket will use only IPv4. When IPv6
       is enabled the socket will connect to, or accept connections from, both
       IPv4 and IPv6 hosts.
```
I could see the argument that explicitly enabling ipv6 on the 0MQ socket, then binding it to an _interface_ that doesn't support ipv6 should be a hard error.  Note that it's not an error to bind to an IPv4 _address_ with ipv6 enabled.

To answer your other question, everything in the `[bootstrap]` stanza only applies to the config bootstrap method, not to the PMI bootstrap method.  Maybe that should be made clearer in the man page update.

Reply:
> To answer your other question, everything in the [bootstrap] stanza only applies to the config bootstrap method, not to the PMI bootstrap method. Maybe that should be made clearer in the man page update.

Thanks, that may not be necessary. I misread the changes to `boot_pmi.c` (or didn't actually look at them) and assumed something in the PMI bootstrap had changed. Excuse my mistake!